ncclient/transport/ssh.py

```diff
@@ -59,7 +59,7 @@
 # * result.group(1) will contain the digit string for a chunk
 # * result.group(2) will be defined if '##' found
 #
-RE_NC11_DELIM = re.compile(r'\n(?:#([0-9]+)|(##))\n')
+RE_NC11_DELIM = re.compile(rb'\n(?:#([0-9]+)|(##))\n')
```

Comment:
This causes a Python 2.7 syntax error. We will be supporting 2,7 for quite a while, so this needs addressed.